index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Hi+Melody&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="index.css" />
+    <title>Awesome books</title>
+
+   
+  </head>
+
+  <body>
```

Comment:
- [OPTIONAL ] Be careful with blank lines, do not add blank lines, or indentations without a reason. [HTML & CSS best practices
](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)